src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,126 @@
+import classNames from 'classnames';
+import { SearchLink } from '../SearchLink';
+
+type Props = {
+  query: string;
+  onQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  centuries: string[];
+  sex: string;
+};
+
+export const PeopleFilters: React.FC<Props> = ({
+  query,
+  onQueryChange,
+  centuries,
+  sex,
```

Comment:
Maybe it's better to get these properties here with useSearchParams, because you set it here?

Reply:
I agree, thnx for reviewing